configure

```diff
@@ -1,8 +1,5 @@
 #!/bin/sh
 # This script was automatically generated by ACR v1.9.4
-# @author: pancake <nopcode.org>
```

Comment:
These fields are generated by acr. Please just regenerate this file.

Reply:
i do not know who to do that.